### PR TITLE
CockroachDB

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR documents the CockroachDB service that is available in PDE.

## Review Changes

💻 [Preview](https://deploy-preview-1226--docs-spectrocloud.netlify.app/devx/app-profile/services/service-listings/cockroach-db)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PPD-914?search_id=53326caa-1b8e-4dc6-9de3-47565aa7b91f)

### CODE REVIEW

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:4]:
```suggestion
metaDescription: "Learn how to use CockroachDB with Palette Dev Engine."
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:19]:
```suggestion
CockroachDB is a [distributed SQL database](https://www.cockroachlabs.com/blog/what-is-distributed-sql/) designed for cloud-native environments. CockroachDB provides a reliable and scalable solution for managing data across multiple nodes and regions. Its architecture automates data replication, sharding, and rebalancing, By simplifying operational tasks, Cockroach enables developers to concentrate on building their applications.
```

Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:19]: It's up to you if you want to use the above suggestion. I offer it only to break up some long sentences.

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:34]:
```suggestion
- A Virtual Cluster with the following minimum resources.
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:50]:
```suggestion
3. Navigate to the left **Main Menu** and click on **App Profiles** to create a new app profile. Review [Create an App Profile](/devx/app-profile/create-app-profile/) for more information. Provide the following basic information and click **Next**.
```

Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:50]: For consistency with other docs.

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:65]: Another line needs to be inserted between Name and Username to make the spacing consistent in the list.

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:76]:
```suggestion
  * **PersistentVolumeClaim Size (GiB)**: Select the volume according to the storage volume available in the cluster group and virtual clusters. Ensure you do not exceed the maximum storage size for your virtual cluster. 
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:91]: I'm okay with using this Validation format, but I thought you wanted numbered steps?

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:87]:
```suggestion
* To verify your database service is in the app profile, navigate to the **App Profiles** page, where all your app profiles are listed. Select the app profile to review the service layers. The next screen displays the different service layers that make up the app profile. Ensure CockroachDB is an available service layer.
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:129]:
```suggestion
#### Example: 
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:140]:
```suggestion
#### Output:
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:146]:
```suggestion
To learn more about developing with CockroachDB, check out the [CockroachDB Developer Guide](https://www.cockroachlabs.com/docs/stable/developer-guide-overview.html). The developer guide is a great resource for understanding how to get started with CockroachDB and build applications that are scalable, resilient, and secure.
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:149]: Maybe we should have a style about spacing Resources? This looks good to me. When they're double spaced, it sometimes looks too spaced. It could be we double space if there are 5 or more resources. Single space 5 or less. I could add this to our team meeting agenda.

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md:92]:
```suggestion
* To verify your database service is in the app profile, navigate to the **App Profiles** page, where all your app profiles are listed. Select the app profile to review the service layers. The next screen displays the different service layers that make up the app profile. Ensure Postgres is an available service layer.
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md:155]: These Resources are more spaced. Our team should probably put our heads together for consistency.

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/4-vault.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/4-vault.md:29]:
```suggestion
Vault is deployed as a single container in the virtual cluster, and the container is not tied to any particular node.
```

**Review by ritawatson — APPROVED** [review]
Some minor comments.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1226#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1226#pullrequestreview-1371941978" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:65]:
```suggestion
  * **Name**: The application name.
  
```

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:91]: You're absolutely right. I forgot to push it up my last change

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1226#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1226#pullrequestreview-1371941978" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1226#pullrequestreview-1373474502" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
